### ISSUE TITLE
Perform a synchronous fetch?

### ISSUE THREAD

**Issue by markst** [Alterplay/APAddressBook#111]
Although a majority of the time I can fetch asynchronously, there one exception where by I need to perform a synchronous fetch for a contact image.

Is this at all possible?

I've tried using `ABAddressBookGetPersonWithRecordID`, but need to pass the encapsulated `ABAddressBookRef`.

**Comment by belkevich** [comment]
Yes, it's possible. I can add synchronous method. But why asynchronous one doesn't fit you?
